indexer/indexer_tests/osm_editor_test.cpp

```diff
@@ -754,6 +764,8 @@ void EditorTest::IsCreatedFeatureTest()
   auto const mwmId = ConstructTestMwm([](TestMwmBuilder & builder)
   {
     builder.Add(TestCafe(m2::PointD(1.0, 1.0), "London Cafe", "en"));
+
+    builder.Add(TestPOI(m2::PointD(10, 10), "Corner Post", "default"));
```

Comment:
And there will be two ways how it could be implemented by TestMwmBuilder:
* create four fake corner features with mangled names, types and ids, as you do
* do it honestly